src/Illuminate/Http/Client/PendingRequest.php

```diff
@@ -89,6 +89,13 @@ class PendingRequest
      */
     protected $stubCallbacks;
 
+    /**
+     * The middleware that will be run as part of this request.
+     *
+     * @var array
```

Comment:
Can we be a little more specific about the type?

Reply:
Added in latest commit. `callable` seems to be the right one since that's what all the helper functions in `GuzzleHttp\Middleware` return.